galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -366,3 +366,12 @@ messages:
         type: string
       - name: gateway
         type: string
+
+  - name: "ServiceEntryProtocolAndAddressesEmpty"
+    code: IST0133
+    level: Warning
+    description: "Addresses and Protocol is Empty in ServiceEntry."
```

Comment:
```suggestion
    description: "Addresses or Protocol is empty in ServiceEntry."
```

Reply:
I prefer with `Addresses and Protocol is empty in ServiceEntry.` since this analyzer match if Addresses and Protocol are empty.

Reply:
I think `Addresses` is optional but the user must set `Protocol`?

Reply:
miss documentation probably, please check @howardjohn comment https://github.com/istio/istio/issues/27990#issuecomment-709593864

Reply:
this is fixed